### DISCUSSION TITLE
Penpot and HTTPS

### DISCUSSION THREAD

**Discussion by vdegrandpre** [uxbox/uxbox#977] [Q&A] [answered]
Hi Community !

I search the frontend repository but I found nothing on that topic.

What would be a good solution to have the containerized application served over HTTPS ?

**Comment by niwinz** [comment, marked as answer]
Hi @vdegrandpre, I'm not clearly understand your question.
There are nothing on penpot side that need to be done to you will be able to serve it under https. Just put a http proxy (nginx, or similar) with correct ssl certificate for your domain, and properly configure the `PENPOT_PUBLIC_URI` option, correctly setting the `https` protocol.

**Reply by vdegrandpre** [reply]
Hi @niwinz, thanks for your quick answer. Is that doable within the docker-compose.yaml file ?

**Reply by niwinz** [reply]
Probably, you just will need probably use an additional container with the proxy server, I probably recommend [caddy](https://hub.docker.com/_/caddy?tab=description) because it provides a easy https setup for the domain (and the dockerhub readme already has examples). But any http proxy server will work. 

TLS/SSL certificates is something related to the domain this is because we can provide a ready to use https. You just need to use your own solution on top of penpot. Probably we will need to an example of how it can be done in our help center.

**Reply by niwinz** [reply]
Opened an [internal issue on taiga](https://tree.taiga.io/project/penpot/issue/1730) for improve the documentation on this.

**Reply by vdegrandpre** [reply]
Very appreciated !

**Comment by smileBeda** [comment]
Can I ask what's wrong with using the inbuilt example of using `penpot_traefik`?
It doesn't need any NGINX or else to run the tool via HTTPS.

Just a DNS (for the A record of the IP), a valid Email and the config file.
While this could (should) be much better documented it is actually elaborated inside the `yaml` file.
